sky/tools/sky_snapshot/BUILD.gn

```diff
@@ -23,4 +23,12 @@ executable("sky_snapshot") {
     "//sky/engine/bindings:snapshot_cc",
     "//third_party/zlib",
   ]
+
+  ldflags = [
```

Comment:
You will need to brace this with a `if (!is_ios && !is_mac)` to fix http://208.52.189.105:8010/builders/build_engine/builds/172/steps/shell/logs/stdio